### ISSUE TITLE
Create Bullseye Dockerfile

### ISSUE THREAD

**Issue by magsol** [magsol/wildlife-cameras#18]
Custom Dockerfile that builds off [Debian 11 (bullseye) base image](https://docker.debian.net/) for the dev environment in #17.

```
FROM debian:11
```

This way we can also install all the annoying customizations we've had to make so far.

**Comment by magsol** [comment]
Closed via 995ee30be2ef42742b9dc3f22c8310cfd3a85ac5

**Comment by magsol** [comment]
Had to deviate in the approach to building the Dockerfile. Turns out, sourcing it directly from debian left out a LOT of prereqs unique to Raspberry Pi variants, enough so that recreating them by trial-and-error would have eventually worked but been very tedious and time consuming.

So instead of using debian as a base image, we switched to balena: 

https://hub.docker.com/r/balenalib/raspberrypi3-debian/

This took care of all the needed prerequisites so we could simply focus on installing all the needed packages for our dev work.
